rotkehlchen/db/upgrades/v34_v35.py

```diff
@@ -108,6 +108,19 @@ def _add_blockchain_column_web3_nodes(cursor: 'DBCursor') -> None:
     cursor.execute('DROP TABLE web3_nodes_old')
 
 
+def _update_ignored_assets_identifiers_to_caip_format(cursor: 'DBCursor') -> None:
+    cursor.execute('SELECT value FROM multisettings WHERE name="ignored_asset";')
+    old_ids_to_caip_ids_mappings: List[Tuple[str, str]] = []
+    for (old_identifier,) in cursor:
+        if old_identifier.startswith(ETHEREUM_DIRECTIVE):
+            old_ids_to_caip_ids_mappings.append((strethaddress_to_identifier(old_identifier[6:]), old_identifier))  # noqa: E501
+
```

Comment:
Also as Lefteris mentioned once, we should be trying to avoid `strethaddress_to_identifier` in favor of `evm_address_to_identifier`

Reply:
Yeah, I remember but the content of the function was updated recently and it's basically a wrapper around `evm_address_to_identifier`. 
I'll use `evm_address_to_identifier`